src/data/proxy/UserProfileStore.java

```diff
@@ -21,23 +21,30 @@ public UserProfileStore() {
     /**
      * Writes a user profile to storage.
      * 
-     * @param profile
+     * @param profile The UserProfile object to store
      */
     public void write(UserProfile profile) {
         userProfiles.put(profile.getId(), profile);
     }
+
+    /**
+     * Remove a user profile from storage.
+     *
+     * @param id The String id for the UserProfile to delete
+     */
+    public void delete(String id) { userProfiles.remove(id); }
```

Comment:
Let's be sure to adhere to a consistent coding standard. Can you please match the format used elsewhere?